docs: add license note for interactive debugger (#2760)
forcedotcom/salesforcedx-vscode: docs/_articles/en/apex/interactive-debugger.md

The first time that you use Apex Debugger in VS Code, complete these setup steps.

> Note that you will need to have the appropriate debugger license. The default license provided to License Management orgs can only be used with the [ISV Customer Debugger](./en/apex/isv-debugger) to debug Subscriber orgs. If you do not have the appropriate license, please consider using the [Replay Debugger](./en/apex/isv-debugger).

Reviewer: ```suggestion
Note: You can use the default license provided to License Management orgs only with the [ISV Customer Debugger](./en/apex/isv-debugger) to debug subscriber orgs. If you don't have an Apex Debugger license, use the [Replay Debugger](./en/apex/isv-debugger).
```

Reviewer: @sfsholden - some things to review so we can get this note right.
https://help.salesforce.com/articleView?id=000314082&type=1&mode=1
https://salesforce.stackexchange.com/questions/214760/salesforce-dx-apex-debugging-license-issue

Reviewer: From: https://developer.salesforce.com/blogs/engineering/2016/01/apex-interactive-debugger.html
You can purchase debugger sessions, which are shared across all users and all sandboxes from your parent org.

If your organization has purchased debugging capability, all of the sandboxes spawned from it are enabled for debugging. The number of sessions you purchase represents the number of your sandboxes that can be debugging at the same time.

Reviewer: So, if you don't have any Apex sessions available, use the Replay Debugger?

Author: @ekapner This looks like a good resource we could use for helping users determine how to get the licenses: https://help.salesforce.com/articleView?id=000314082&type=1&mode=1

Author: > So, if you don't have any Apex sessions available, use the Replay Debugger?

Do we want to add this? I'm not sure personally. I wanted to include the section about the Replay Debugger because it's such a good tool (and free). For this specific use case, where the customer was trying to use the interactive debugger from an LMO, the replay debugger was really the better fit.
